libs/net/nodenetworkconfigurationpolicy.py

```diff
@@ -0,0 +1,159 @@
+# API: https://nmstate.io/devel/yaml_api.html#dhcp
+from dataclasses import asdict, dataclass
+from typing import Any
+
+from ocp_resources.exceptions import NNCPConfigurationFailed
+from ocp_resources.node_network_configuration_policy_latest import NodeNetworkConfigurationPolicy
+from ocp_resources.resource import Resource, ResourceEditor
+from timeout_sampler import retry
+
+WAIT_FOR_STATUS_TIMEOUT = 90
+WAIT_FOR_STATUS_SLEEP = 5
+
+
+@dataclass
+class STP:
```

Comment:
Please arrange all classes here from top to bottom per their reference.
At the top, place `DesiredState` and from there per their order.

I will like to read this from top as a story.